torch/ao/quantization/fx/backend_config/native.py

```diff
@@ -91,6 +93,15 @@ def _get_default_op_backend_config(op, dtype_configs):
     ],
 }
 
+_HARDSIGMOID_MODULE_CONFIG = {
+    "pattern": torch.nn.Hardsigmoid,
+    "observation_type": ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT,
+    "overwrite_output_fake_quantizer": FixedQParamsFakeQuantize.with_args(observer=default_affine_fixed_qparams_observer),
```

Comment:
hmmm, currently we override the observer for these ops, so we'd need to put this into config I think, but maybe we could do a refactor later to move these to default qconfig_dict setting, and put the requirement here, like "quant_min", "quant_max", "dtype" requirement etc. but to unblock this refactor I think we can land this first

Reply:
would it be worth considering not landing this then?  Seems like currently this happens in the workflow, which is not ideal.  However, putting it into the "wrong" config sounds even more confusing than leaving it hardcoded in the code.

Reply:
Maybe I can start with `_` for now for the configuration key? we still need to do this refactor and use backend_config_dict everywhere in the prepare first, fixing the existing issue will be a separate step I think

Reply:
to finish the frontend refactor, we don't have the option to just leave it hardcoded in the current state

Reply:
makes sense, how about moving this to qconfig_dict right away then instead of putting it here?  What's the eng cost on that?

Reply:
unfortunately this is not possible to currently, since the new path is not fully enabled yet in the code, current state is kind of intermediate state where we use the default quant patterns and the new quant pattern together see L138 and L144 in https://github.com/pytorch/pytorch/blob/0765a804911673fb2d9694a76ba0196ea0eddec4/torch/ao/quantization/fx/backend_config/utils.py#L133, and the backend_config_dict is actually None for fbgemm/qnnpack currently, which will be used in https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fx/prepare.py#L181 to guard the configurations(dtype, quant_min, quant_max etc.) in QConfig, this path can only be enabled after we have everything fully migrated to backend_config_dict.

It's a little bit complicated, but the short answer is we can't do that right now

Reply:
yeah sure